app/assets/stylesheets/shared/variables/_layout.scss

```diff
@@ -0,0 +1,6 @@
+// Breakpoints
+$desktop_breakpoint: 1024px;
+$tablet_breakpoint: 768px;
+$phablet_breakpoint: 640px;
+$mobile_breakpoint: 480px;
```

Comment:
@sauloperez Context about why I don't use [existing breakpoint mixing](https://github.com/openfoodfoundation/openfoodnetwork/commit/035bde9effe9cb55b2d95e5181c8b9ed615967cc#r42455370)

This is a compromise because now variables are shared :)

Reply:
nice, I'd even create new breakpoints with min-width.